.changeset/rude-mayflies-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-vault-backend': patch
+---
+
+Added errorHandler() middleware to vault-backend to prevent errors to cause a crash
```

Comment:
just nitpicking - optional suggestion

```suggestion
Added `errorHandler()` middleware to `router` to prevent crashes caused by fatal errors in plugin backend
```

`vault-backend` context is clear here as it would be listed in the changeset section for the `@backstage/plugin-vault-backend`